app/models/session_activation.rb

```diff
@@ -53,4 +59,22 @@ def exclusive(id)
       where('session_id != ?', id).destroy_all
     end
   end
+
+  private
+
+  def assign_user_agent
+    self.user_agent = '' if user_agent.nil?
+  end
+
+  def assign_access_token
+    superapp = Doorkeeper::Application.find_by(superapp: true)
+
+    return if superapp.nil?
```

Comment:
Since this check hasn't existed previously, it will breaks instances which don't have "Web" application. For example, Docker-Guide says you should run docker-compose run --rm web rake db:migrate, but db:migrate doesn't call db:seed which contains "Web" app generation.

Anyway, super app must be existent, so we shouldn't ignore non existence of it I think.